sql_playground/playground.sql

```diff
@@ -6,9 +6,10 @@
 -- users
 -- topics
 
-SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count
-FROM comments
-RIGHT JOIN articles
-ON comments.article_id = articles.article_id
-GROUP BY articles.article_id
-ORDER BY created_at DESC;
+INSERT INTO comments
+(body, article_id, author)
+VALUES
+('Maybe, like a cat you''ve seen something minute on the wall.', 
+11, 
+'butter_bridge')
```

Comment:
Please make sure these sql_playground files are not included when you merge to main. You could add them to your gitignore perhaps?